app/2024/community-summit/page.jsx

```diff
@@ -0,0 +1,203 @@
+import Link from "next/link";
+
+
+export default function CommunitySummit() {
+  return (
+    <>
+      <section className="flex bg-cyan-900 text-zinc-300 justify-center">
+        <div className="flex py-10 lg:items-center">
+          <div className="md:mx-10 text-center mx-4">
+            <h1 className="text-3xl font-extrabold sm:text-5xl text-white">
+              Python Community Summit Schedule
+            </h1>
+            <p className="mt-4 sm:text-xl mx-5">
+              Come take part in amazing community events
+            </p>
+            <p className="mt-4 sm:text-xl mx-5 font-bold">
+              Date: 9th October 2024
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section className="mx-8 md:mx-28 max-w-screen-xl my-10 text-justify text-lg">
+
+        <div class="container mx-auto">
+          <table class="border-separate w-full mt-4" style={{ borderSpacing: '0.5rem' }}>
+            <thead>
+              <tr>
+                <th class="p-4 bg-white rounded shadow">Time</th>
+                <th class="p-4 bg-white rounded shadow">Activity</th>
+                <th class="p-4 bg-white rounded shadow">Facilitator</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td class="rounded shadow bg-cyan-700">
+                  <div class="rounded text-center text-white p-2">
+                    <h4>10:00 to 11:00</h4>
+                  </div>
+                </td>
+                <td class="rounded shadow bg-white">
+                  <div class="p-2 text-black">
+                    <h4>Welcome remarks and Introductions</h4>
+                  </div>
+                </td>
+                <td class="rounded shadow bg-white">
+                  <div class="p-2 text-black">
+                    <h4>Jerry Kakoza</h4>
```

Comment:
`Jerry Kakooza` is apparently not available. there is a lady called `Deborah` taking up the place

Reply:
I will make the necesary changes.

Reply:
okay